### ISSUE TITLE
[FEATURE SUGGESTION] Filter reposts from Stream

### ISSUE THREAD

**Issue by daniellbg** [Soundnode/soundnode-app#736]
OS X v0.6.4
This is a common request by the community using the standard web client as well. 
Reposts are clogging up the Stream if you are following a large number of people, since you usually are following someone for their own music, not the music they in turn like/promote.

**Comment by Chriz007** [comment]
Exactly. This has kind of turned me off from using soundcloud or this app because of unnecessary spam

**Comment by weblancaster** [comment]
I have been pretty busy but I have that on my todo's.. I even had implemented at some point but I wasn't happy with the design and removed.

**Comment by Pitros** [comment]
Just a quick question, removing track reposts it's something that you would turn on and woudn't turn off (toggle in settings) or it would be nice to have quick access to toggle that - let's say from stream view?

**Comment by daniellbg** [comment]
In my case, definitely a quick toggle from stream. I mostly find reposts annoying but it serves a purpose at times if I feel like broadening my horizons outside of my comfort zone.

**Comment by dgattey** [comment]
Toggle from stream or Top 50 would be nicest.
